### ISSUE TITLE
Battery loading problem

### ISSUE THREAD

**Issue by Lueddi** [fhorinek/SkyDrop#189]
Hello After my Display issue, I bought the New Version directly from the Skybean Shop. 
Now I habe got the problem,  that when I charge the skydrop,    it is more empty than before the charging. The LED is Green and it says full.  But it seems that the Skydrop goes on After it is charged and will drain the battery. (the usb cable is still connected) So first the percentage raises up to 95% (i checked) but when I didn't disconect the usb cable the skybean will Run in the Backround. 

Please give me an advice to solve this issue. 

Thanks!

**Comment by rrunner911** [comment]
Exactly the same issue i found out. Seems that the charging is stopped in the background, while the device is on and looses battery. Hope fix comes soon!

**Comment by fhorinek** [comment]
@Lueddi @rrunner911 we are talking about brand new devices in both cases?

**Comment by rrunner911** [comment]
Yes, mine ist Skydrop new generation. Ordernumber 374

**Comment by Lueddi** [comment]
Ye, mine too, I order es it via E-Mail, on 7. Mai 2016.  [#681414]

**Comment by Lueddi** [comment]
Yes, mine is also the Skydrop New Generation. I ordered via Email on 7.Mai 2016 [#681414]

**Comment by Lueddi** [comment]
Are there any recommended solutions for this problem?

**Comment by fhorinek** [comment]
We are working on reliable method of battery percentage conversion right now. Currently it works fine on 90% or more devices. We just have to figure it out. It it not a HW problem. The batteries have the same capacity

**Comment by Lueddi** [comment]
Hello fhorinek, 

Do you have got any further Information for us? 
Thank you!

**Comment by fhorinek** [comment]
sorry, just upgrade to newest firmware, there is battery calibration method implemented
